doc/whats-new.rst

```diff
@@ -107,10 +138,13 @@ Internal Changes
 - Move ``parallelcompat`` and ``chunk managers`` modules from ``xarray/core`` to
   ``xarray/namedarray``. (:pull:`8319`)
   By `Tom Nicholas <https://github.com/TomNicholas>`_ and `Anderson Banihirwe <https://github.com/andersy005>`_.
-- Imports ``datatree`` repository and history into internal location. (:pull:`8688`)
-  By `Matt Savoie <https://github.com/flamingbear>`_ and `Justus Magin <https://github.com/keewis>`_.
-- Adds :py:func:`open_datatree` into ``xarray/backends`` (:pull:`8697`)
-  By `Matt Savoie <https://github.com/flamingbear>`_.
+- Imports ``datatree`` repository and history into internal
```

Comment:
Is this allowed? (updating released changes)

Reply:
If it's a correction / clarification then sure? I don't think we have a formal rule

Reply:
this should be fine, as long as it's just an update (for anything new I'd recommend a new entry).